api-docs/rst/dev-guide/concepts.rst

```diff
@@ -61,40 +68,65 @@ Scaling groups
 
 The scaling group is at the heart of an Autoscale deployment. The
 scaling group specifies the basic elements of the Autoscale
-configuration. It manages how many servers can participate in the
-scaling group. It also specifies information related to load balancers
-if your configuration uses a load balancer.
+configuration. It manages how many scaling units can participate in
+the scaling group. It also specifies information related to load
+balancers if your configuration uses a load balancer.
 
```

Comment:
The scaling group specifies the basic elements of the Autoscale configuration and manages the number of servers that can be included in the scaling group. If your configuration uses a load balancer, the scaling group also specifies information related to managing load balancers. 

Revision note:    My revision might not be technical accurate. Problem that I'm trying to solve:  First sentence of paragraph indicates that the topic is "scaling groups" .  The subsequent sentences have vague pronoun reference "It".  I replaced that with "Scaling groups"     Not quite sure about "participate in"   changed to included ....